include/bouncer.h

```diff
@@ -516,7 +516,6 @@ struct PgPool {
  * that use these credentials.
  */
 struct PgCredentials {
-	struct List pool_list;		/* list of pools where pool->user == this user */
```

Comment:
Comments related to `pool_list` should be moved to `PgGlobalUser`.

Reply:
I fixed the comment. The previous comment was also incorrect now, because this PR starts to always share the pool_list.